### ISSUE TITLE
Broken syntax after ticks ``` within @moduledoc string

### ISSUE THREAD

**Issue by macabl** [elixir-editors/emacs-elixir#444]
![sel-2019-10-11--19:16:16](https://user-images.githubusercontent.com/48124288/66671251-e2a2a600-ec5b-11e9-9551-c6ecad11d88f.png)

**Comment by jsmestad** [comment]
Are backticks valid like that in moduledoc? I know docs can be markdown, but I was unsure if that meant also codeblocks and code highlighting with like ` ```json `

**Comment by axelson** [comment]
Is this still an issue? I'm not able to reproduce it. ` ```json ` should be valid within a heredoc (`"""`), although highlighting the content as JSON is not supported.

**Comment by victorolinasc** [comment]
I believe this is a side effect of the many changes that went into Emacs 27 about regexes. Currently, elixir-mode.el is almost completely based on regexes (as most major-modes are). 

I had similar issues while surfing the master branch of Emacs. I am currently on Emacs 28 on feat/native-comp branch and also can't reproduce the results.

We had other issues about regexes mentioned here and some were even discussed on emacs-devel. I think we should wait for a reply from who posted the issue but I guess it was not anything in the code of elixir-mode.

**Comment by axelson** [comment]
Yeah, I was having issues with `"""` on Emacs 27, but for me that went away a few weeks ago (and I'm still on Emacs 27).

**Comment by konstantinzolotarev** [comment]
I suppose this is issue in JSON itself (missing comma after `fronts#index`).

**Comment by victorolinasc** [comment]
@macabl can you confirm the issue is gone? If not, can you detail your setup a little bit more? I think there are people using modes for highlighting documentations that might be the cause here too.
